src/my_functions_product1.py

```diff
@@ -0,0 +1,721 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Jun 11 15:15:42 2018
+
+@author: bursaliogluozgun
+"""
+import pandas as pd
+import numpy as np
+
+
+import keras as ke
+import keras.backend as K
+from keras.layers import Input, Dense, Dropout
+from keras.models import Model
+from keras.models import load_model
+# model reconstruction from JSON:
+from keras.models import model_from_json
+
+
+import sklearn as sk
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import StandardScaler
+from sklearn.metrics import accuracy_score, roc_auc_score
+from sklearn.utils.class_weight import compute_class_weight
+#==============================================================================
+def Input_file_checking(filename):
+    """ 
+       This function checks if the input file exists
+    """
+    data_file_error = False
+
+    
+    
+    try:
+        #open and closing the log file
+        fdata = open(filename,'r')
+        fdata.close() 
+    except IOError:
+        print('Cannot open '+filename)
+        data_file_error = True
+        
+        
+#==============================================================================
+def reading_data(path):
+    your_data_df = pd.read_csv(path)
+    
+    #your_data_df['Prediction'] = your_data_df[' <=50K']
+    
+    all_columns = np.array(your_data_df.columns)
+    print('Columns in your data are:\n ',all_columns)
+    all_columns = set(all_columns.tolist())
```

Comment:
I believe the set() over here will not do anything. If multiple columns have the same header, they will be handled by default (https://pandas.pydata.org/pandas-docs/stable/generated/pandas.read_csv.html, param: mangle_dupe_cols, default: True)